### ISSUE TITLE
Converter show error - where are logs?

### ISSUE THREAD

**Issue by JustAPhrog** [oliverschwendener/ueli#1042]
Hello, I've installed ueli (v8.24.1) on windows 11. I tyied to use Converter but I received `An error occurred`. I don't know where find logs - nothing is written on readme. Anybody?

**Comment by oliverschwendener** [comment]
You can open the debug log in the settings:

![image](https://github.com/oliverschwendener/ueli/assets/15727229/b6b1c537-9b71-47c2-b8d2-b337ddfe11a4)

**Comment by SebastianSava** [comment]
I have the same error.

This is in my `debug.log` file:

```
2023-09-29T06:57:20.172Z error: Unable to get exchange rate. Result: [object Object]
```

**Comment by tkohlmeier** [comment]
Seems like exchangerate.host now requires an API key:

    {
      "success": false,
      "error": {
        "code": 101,
        "type": "missing_access_key",
        "info": "You have not supplied an API Access Key. [Required format: access_key=YOUR_ACCESS_KEY]"
      }
    }

There is a free plan, but it is limited to 1,000 requests per month ([source](https://exchangerate.host/product)). So it looks like we need to change the source again.

**Comment by wywywywy** [comment]
There are 2 options I think.

1. Ask user to provide their own API key (free to register anyway)
2. Use https://github.com/fawazahmed0/currency-api and get free & fast exchange rates forever! As long as @fawazahmed0 keeps up with the maintenance that is

**Comment by wywywywy** [comment]
I just created a new PR to fix this #1043
